app/containers/FileModal.js

```diff
@@ -38,6 +44,23 @@ const styles = StyleSheet.create({
 	},
 	indicator: {
 		flex: 1
+	},
+	video: {
```

Comment:
No need to set width and height.
`flex: 1` works nicely for both portrait and landscape.

Tip: try not to use `Dimensions` directly on styles.
In this case, it works ok for portrait, but try to change your device orientation... just for fun ;)

Reply:
lol, I did try to rotate it - doesn't look good :)